backend/src/services/base.service.ts

```diff
@@ -98,4 +105,30 @@ export abstract class BaseService<T, C, U> {
   async getAll(): Promise<T[]> {
     return this.model.find();
   }
+
+  /**
+   * Appends the uploaded file to a document with given ID.
+   *
+   * @param _file - The file to append.
+   * @param _documentID - The ID of the document to which to append the file
+   * @param _fileType - Type of the uploaded file.
+   * @returns A promise that resolves to the updated entity.
+   */
+  abstract appendFile(
```

Comment:
i think this sould be inside a interface. Btw. we have to rename baseservice to crud-service, since we will have services without crud operation. my bad

Reply:
I do not think it should be an interface. The abstract function requires the children to implement that function, which is what we want for our use case. Renamed.

Reply:
Deleted the declaration from crud service.